code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -267,6 +267,7 @@
 	new /obj/item/modular_computer/tablet/syndicate_contract_uplink/preset/uplink(src)
 	new /obj/item/storage/box/syndicate/contractor_loadout(src)
 	new /obj/item/melee/classic_baton/telescopic/contractor_baton(src)
+	new /obj/item/storage/box/syndie_kit/swapper_implant
```

Comment:
Not here please :sweat_smile:, the `item_list` underneath. `item_list` gets randomly chosen from while here would be a permanent item.